recipes/wmctrl/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "wmctrl" %}
+{% set version = "0.3" %}
+{% set sha256 = "d806f65ac1554366b6e31d29d7be2e8893996c0acbb2824bbf2b1f49cf628a13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - wmctrl
+
+about:
+  home: https://bitbucket.org/antocuni/wmctrl
+  license: BSD
+  license_family: BSD
+  summary: 'A tool to programmatically control windows inside X'
+  doc_url: https://bitbucket.org/antocuni/wmctrl
```

Comment:
It doesn't look like there is any documentation at this link or others so please delete the `doc_url`.